src/main/server.ts

```diff
@@ -1,17 +1,27 @@
 #!/usr/bin/env node
-const { Logger } = require('@hmcts/nodejs-logging')
+const appInsights = require('applicationinsights');
```

Comment:
`yarn add @types/applicationinsights` will add type defs and you can use import same like other imports.

Reply:
done